src/jit/utils.cpp

```diff
@@ -353,7 +353,7 @@ void                dspRegMask(regMaskTP regMask, size_t minSiz)
 #endif // _TARGET_*
             }
             // We've already printed a register. Is this the end of a range?
-            else if ((regNum == REG_INT_LAST)
+            else if (regNum == REG_INT_LAST
```

Comment:
I must also admit that this double paren convention that clang pushes here is kind of strange. At the other hand, bugs like assignment in a condition instead of a comparison are easy to make, so the convention of using two pairs of parens as an indicator that the developer really meant to assign there kind of make sense.